mutiny-core/src/networking/proxy.rs

```diff
@@ -123,18 +119,14 @@ pub fn tcp_proxy_to_url(proxy_url: &str, peer_addr: &str) -> Result<String, Muti
     ))
 }
 
-pub fn mutiny_conn_proxy_to_url(proxy_url: &str, peer_pubkey: &str) -> String {
-    format!("{proxy_url}/v1/mutiny/{peer_pubkey}",)
-}
-
 #[cfg(test)]
 mod tests {
     #[cfg(feature = "ignored_tests")]
-    use crate::proxy::*;
+    use crate::networking::proxy::*;
 
     use crate::test_utils::*;
 
-    use crate::proxy::{mutiny_conn_proxy_to_url, tcp_proxy_to_url};
+    use crate::networking::proxy::tcp_proxy_to_url;
 
     use wasm_bindgen_test::{wasm_bindgen_test as test, wasm_bindgen_test_configure};
 
```

Comment:
should put `PEER_PUBKEY` behind `#[cfg(feature = "ignored_tests")]` to get rid of the warning.